app/views/customers/show.html.erb

```diff
@@ -0,0 +1,19 @@
+<h1><%= @customer.name %></h1>
+
+<h2>Items</h2>
+<ul class="items">
+  <% @customer.items.each do |item| %>
+    <li>
```

Comment:
For my suggestion about testing a within block for each individual item, you could have a css id tied to this <li> element that changes dynamically with the item.id. Allows for more targeted testing.